server/src/core/payment/flutterwave-payment-provider.ts

```diff
@@ -161,8 +161,30 @@ export class FlutterwavePaymentProvider implements PaymentProvider {
       throw createFlutterwaveApiError(e.response.data && e.response.data.message || e.message);
     }
   }
-  sendFundsToUser(user: User, amount: number, metadata: any): Promise<SendFundsResult> {
-    throw new Error('Method not implemented.');
+  async sendFundsToUser(user: User, amount: number, metadata: any): Promise<SendFundsResult> {
+    const transferArgs = { 
+      account_bank: 'MPS',
+      account_number: user.phone,
```

Comment:
I believe the phone number passed to flutterwave should be in the 07... format. So you'd need to convert from the social relief format (2547...) to 07... that's how we handled it in other methods of this class.